bindings/java/src/main/java/org/apache/opendal/enums/Schema.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.opendal.enums;
+
+public enum Schema {
```

Comment:
Can we remove this change? It's another new API to our users which is not expected in this PR. It's ok to use plain string in the tests.

Reply:
I think adding a schema enum would be a good way to visualize what services we currently support and to standardize user input. I can create a new pr to submit this if you agree.

Reply:
> visualize what services we currently support

The services we support are determined by how the Java binding is constructed. Sharing the entire list here doesn't seem like a good idea to me. I prefer to remove it for now.

Reply:
OK, I'll remove it.